python/qa_tools/hid.py

```diff
@@ -0,0 +1,37 @@
+#
+# hid.py
+#
+# Copyright (C) 2019 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPLv2
+#
+# Access to qa-hid keyboard simulator functions
+#
+
+import os
+
+qa_hid='sudo /usr/bin/qa-hid'
+
+
+def call_hid(command_line):
```

Comment:
```suggestion
def _call_hid(command_line):
```
I'd recommend prefixing this function with an underscore to explicitly say this is function is internal to the module.